legal-api/src/legal_api/services/authz.py

```diff
@@ -398,7 +431,7 @@ def get_allowed_filings(business: Business, state: Business.State, legal_type: s
     return allowable_filing_types
 
 
-def has_blocker(business: Business, state_filing: Filing, allowable_filing: dict, has_default_blocker: bool):
+def has_blocker(business: Business, state_filing: Filing, allowable_filing: dict, has_default_blocker: dict):
```

Comment:
Maybe rename this (and others below) if it no longer returns a boolean (re: has vs has not). How about `get_blocker` ?

Reply:
never mind

closing

Reply:
Well... maybe `has_default_blocker` should be `default_blocker` since it's no longer a bool?

Reply:
please rename `has_blocker_default` to `has_business_blocker_dict`

Reply:
yes